### DISCUSSION TITLE
Change location ?!

### DISCUSSION THREAD

**Discussion by cmoisei** [ScoopInstaller/Scoop#4716] [Ideas]
Hello, 

is there a quick way to change the location ?

The main issue I see is that all the shims use full paths and not %SCOOP%/apps/XYZ/current/bin for example.
I setting the $snv:SCOOP to a new location should in principle do the trick but I have to go modify all my paths and user env. vars to the new location. 

Maybe this is a good enhancement request.

Drop me a note  if you have some info for me.
Thanks!

**Comment by rashil2000** [comment]
There is some info here https://github.com/ScoopInstaller/Scoop/issues/3852. But it's not fully supported, being tracked here - https://github.com/ScoopInstaller/Scoop/issues/4573
